deployment/utils/gcb-discord-notifications/index.js

```diff
@@ -0,0 +1,96 @@
+// This is intended to be deployed as a Google Cloud Function to relay
+// Google Build events to Discord
+
+const octokit = require('@octokit/rest')()
+const Discord = require('discord.js')
+
+const hook = new Discord.WebhookClient(
+  process.env.WEBHOOK_ID,
+  process.env.WEBHOOK_TOKEN
+);
+
+// Subscribe is the main function called by Cloud Functions.
+module.exports.subscribe = async (event, callback) => {
+  const build = eventToBuild(event.data);
+
+  const status = ['WORKING', 'SUCCESS', 'FAILURE', 'INTERNAL_ERROR', 'TIMEOUT'];
+  if (status.indexOf(build.status) === -1) {
+    return;
+  }
+
+  // Send message to Discord.
+  const webhook = await createDiscordWebhook(build);
+  hook.send(webhook.message, webhook.options).then(callback);
+};
+
+// eventToBuild transforms pubsub event message to a build object.
+const eventToBuild = (data) => {
+  return data;
```

Comment:
is data b64 encoded or not ? delete one of the two return statement :)

Reply:
Good catch! This was me debugging. :grin: